tox.ini

```diff
@@ -9,6 +9,12 @@ deps =
 commands =
     py.test --doctest-modules --cov=scrapy_crawlera {posargs:scrapy_crawlera tests}
 
+[testenv:security]
+deps =
+    bandit
+commands =
+    bandit -r {posargs:scrapy_crawlera}
```

Comment:
looks this is not running over all the code, when changing to `bandit -r .` the results are:
```
Code scanned:
	Total lines of code: 558399
	Total lines skipped (#nosec): 8

Run metrics:
	Total issues (by severity):
		Undefined: 0.0
		Low: 1619.0
		Medium: 240.0
		High: 42.0
	Total issues (by confidence):
		Undefined: 0.0
		Low: 1.0
		Medium: 31.0
		High: 1869.0
```

Reply:
Also needs to add to the TravisCI to run

Reply:
About the target directory to test, I chose `scrapy_crawlera` because I don’t expect security issues in tests to be relevant.

Reply:
awesome